Make link text for components and patterns consistent (#4278)
alphagov/govuk-design-system: src/patterns/complete-multiple-tasks/index.md

This can be helpful when a task involves:

- some questions that are optional
- writing a long answer (such as in a [Textarea component](/components/textarea/))
- looking up information, such as details about previous jobs
- answers that need to be checked carefully with someone else

> Do this by asking a radio question at the end of the task – either as the last question (if the task is a single page) or on the [Check answers page in your service](/patterns/check-answers/) (if the task uses multiple [Question pages in your service](/patterns/question-pages/)).

Reviewer: I think these are less likely to be confusing, so simplified the link text and removed the link to Question pages, which seems slightly superfluous.

```suggestion
Do this by asking a radio question at the end of the task – either as the last question (if the task is a single page) or on the [Check answers page](/patterns/check-answers/) (if the task involves more than one pages).
```

#### Error messages

If the user does not select an option, show an [Error message component](/components/error-message/) to say: 'Select whether you’ve completed this section'.

{{ example({ group: "patterns", item: "complete-multiple-tasks", example: "have-you-completed-this-section-error", html: true, nunjucks: true, open: false }) }}

## Research on this pattern

This pattern was previously named ‘Task list’ and was [developed by a team at the Government Digital Service (GDS)](https://designnotes.blog.gov.uk/2017/04/04/weve-published-the-task-list-pattern/).

It was then iterated by a cross-government collaboration and published as a new [Task list component](/components/task-list/) with updated guidance and research.

> See the [research on the new Task list component](/components/task-list#research-on-this-component) for details of research done, and known issues and gaps.

Reviewer: ```suggestion
See the [research on the Task list component](/components/task-list#research-on-this-component) for details of research done, and known issues and gaps.
```
